hardware/include/iob_s_if.vh

```diff
@@ -0,0 +1,8 @@
+    //Slave i/f - Native (do not remove indentation)
```

Comment:
File exists

Reply:
Signals and parameters names are not the same